LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 numel007
```

Comment:
Should this include all of us?

Reply:
Good catch. When I completed the document, I was thinking since you are the "owner" of the repo and code, it makes sense to put your username (alternatively you could use your full real name).

All the other people are just contributors to this open source project, but you are the one who owns it.

Reply:
Another option is to create a Github Organization (for reference: https://github.com/softhausHQ), where we can put all the code and split the codebase into 2 repos - backend api & website. However, it's a bit of a pain to do this if it's only going to be for the purpose of 1 project.

We can discuss this topic more in depth during our retrospective.